JavaImportTool/src/main/java/org/envision/java/importtool/ASTConverter.java

```diff
@@ -23,7 +23,7 @@
  ** OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  **
  **********************************************************************************************************************/
-package javaImportTool;
+package org.envision.java.importtool;
```

Comment:
Hmm, we definitely should not use `org.envision` since that domain exists and is not ours. How important is it to have a proper domain? Can we skip the `org` part and just start with `envision`?